src/index.js

```diff
@@ -63,8 +63,8 @@ class Masq {
     this._startReplication()
   }
 
-  disconnect () {
-    return new Promise((resolve) => {
+  async disconnect () {
+    await new Promise((resolve) => {
```

Comment:
why wrap this logic inside a promise ?

Reply:
To get a Promise that resolves after the replication swarm is closed and the session info is deleted (session storage cleared).